rancher2/data_source_rancher2_global_dns_provider.go

```diff
@@ -0,0 +1,57 @@
+package rancher2
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceRancher2GlobalDNSProvider() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceRancher2GlobalDNSProviderRead,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
Datasource schema is incomplete. If using `flattenGlobalDNSProvider` schema should fit rancher2_global_dns_provider resource but computed fields.

Reply:
I updated this to what I think you meant. Can you verify?